packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -1454,29 +1483,39 @@ export class ProjectModel {
                 dashboardVersionIds,
             );
 
-            Logger.debug(
+            Logger.info(
                 `Duplicating ${dashboardTiles.length} dashboard tiles on ${previewProjectUuid}`,
             );
 
             const dashboardTileUuids = dashboardTiles.map(
                 (dv) => dv.dashboard_tile_uuid,
             );
 
-            Logger.debug(
+            Logger.info(
                 `Updating ${chartsInDashboards.length} charts in dashboards`,
             );
             // Update chart in dashboards with new dashboardUuids
-            await newChartsInDashboards.map(async (chart) => {
-                const newDashboardUuid = dashboardMapping.find(
-                    (m) => m.uuid === chart.dashboard_uuid,
-                )?.newUuid;
-
-                return trx('saved_queries')
-                    .update({
-                        dashboard_uuid: newDashboardUuid,
-                    })
-                    .where('saved_query_id', chart.saved_query_id);
-            });
+            const updateChartInDashboards = newChartsInDashboards.map(
+                (chart) => {
+                    const newDashboardUuid = dashboardMapping.find(
+                        (m) => m.uuid === chart.dashboard_uuid,
+                    )?.newUuid;
+
+                    if (!newDashboardUuid) {
```

Comment:
Fixes this bug 

![Screenshot from 2024-04-03 09-29-01](https://github.com/lightdash/lightdash/assets/1983672/513ff4e4-397c-4383-b0b6-ed74d2b1cc81)